src/site/content/en/blog/badging-api/index.md

```diff
@@ -86,67 +85,63 @@ Examples of sites that may use this API include:
 4. Click the **Set** or **Clear** button to set or clear the badge from the app
    icon. You can also provide a number for the *Badge value*.
 
-{% Aside %}
-  While the Badging API *in Chrome* requires an installed app
-  with an icon that can actually be badged, you shouldn't
-  make calls to the Badging API dependent on the install state.
-  The Badging API can apply to *anywhere* a browser might want to show a badge,
-  so developers shouldn't make any assumptions about situations where
-  the browser will display them. Just call the API when it exists.
-  If it works, it works. If not, it simply doesn't.
-{% endAside %}
-
 ## How to use the Badging API {: #use }
 
-Starting in Chrome 73, the Badging API is available as an origin trial
-for Windows (7+) and macOS.
+The Badging API is available as an origin trial in Chrome 79 and later.
 
-{% Aside 'caution' %}
-  Android is not supported because it requires you to show a notification,
-  though this may change in the future. Chrome OS support is pending
-  implementation of badging on the platform.
-{% endAside %}
+The Badging API works on Windows, and macOS. Support for Chrome OS is in
+development and will be available in a future release of Chrome.
+Unfortunately, Android is not supported because it requires you to show a
+notification, though this may change in the future.
```

Comment:
I'm a little confused by this. Notifications seem like they're orthogonal to whether Android lets me alter the appearance of the status bar icons.

Reply:
It isn't ideal but on Android, you have no control over the badges displayed in the launcher.

Some launchers may choose to display the number of unread notifications as a badge on the app icon, but apps cannot control the number directly.

(I'm assuming you meant the launcher icons, not the status bar icons)

Reply:
I'm having trouble with this, specifically:

>Android is not supported because it requires you to show a
notification

Huh?

Reply:
Badges on the Android homescreen

![Android Badges](https://raw.githubusercontent.com/WICG/badging/master/images/android-badge.png)

are not independent of your unread notifications whereas the badging API aims to be. For example, to achieve Badge.set(99), we would have to send the user 99 notifications (which doesn't seem reasonable).

In the above image, the user must have at least one notification for both Twitter and Fitbit in their notification center.

**Note:** Some launchers on Android display counts instead of just a dot.

Reply:
Obviously this isn't the place to explain that complexity. An explanation that is vague is not helpful either. I would remove the explanation and simply say that Android is not supported but we hope it will in the future. Word it how you like.

Reply:
Agree with @jpmedley .

Well, I think saying "Android is not supported" is a little misleading since Android PWAs *already* have badges, if they show a notification. Essentially, this is WAI on Android: you don't get explicit control over your badges, they just come for free with notifications.

How about:

> On Android, the Badging API is not supported. Instead, Android automatically shows a badge on a web app when there is an unread notification, just like for native apps.

Reply:
Done!